internal/prompt/prompt.go

```diff
@@ -28,6 +28,14 @@ func NewDeleteConfirm(entry string) survey.Prompt {
 	return prompt
 }
 
+// NewCustomizeConfirm creates a prompt to confirm if the entry should be deleted
+func NewCustomizeConfirm(message, entry string) survey.Prompt {
+	prompt := &survey.Confirm{
+		Message: fmt.Sprintf("%s %s", message, entry),
+	}
+	return prompt
+}
```

Comment:
[nit] let's make this `NewConfirm(message string)` and you can do the `fmt.Sprintf("%s %s", message, entry)` before calling the function